Source/objects.cpp

```diff
@@ -4519,70 +4519,56 @@ void BreakObject(int pnum, int oi)
 
 void SyncBreakObj(int pnum, int oi)
 {
-	int v2; // eax
-
-	v2 = object[oi]._otype;
-	if (v2 >= OBJ_BARREL && v2 <= OBJ_BARRELEX)
+	if (object[oi]._otype >= OBJ_BARREL && object[oi]._otype <= OBJ_BARRELEX)
 		BreakBarrel(pnum, oi, 0, 1u, 0);

```

Comment:
```suggestion
		BreakBarrel(pnum, oi, 0, 1, 0);
```

Reply:
Thanks! Looks like it even should be `TRUE` now actually.